templates/template.py

```diff
@@ -116,12 +116,14 @@ class NewDataset(datasets.GeneratorBasedBuilder):
     # ds_bigbio = datasets.load_dataset('my_dataset', name='bigbio', data_dir="/path/to/data/files")
 
     # TODO: For each dataset, implement Config for Source and BigBio;
-    #  if dataset contains more than one subset (see examples/bioasq.py) implement for EACH of them. Each of them should contain:
-    #  name: should be unique for each dataset config eg. bioasq10b_(source|bigbio)_[bigbioschema_name]
-    #  version: option = (SOURCE_VERSION |BIGBIO_VERSION)
-    #  description: one line description for the dataset
-    #  schema: options = (source|bigbio_[schema_name]) [schema_name] =(kb,pairs, qa, text, test_to_text, entailment)
-    #  subset_id: subset id is the canonical name for the dataset (eg. bioasq10b)
+    #  If dataset contains more than one subset (see examples/bioasq.py) implement for EACH of them.
+    #  Each of them should contain:
+    #   - name: should be unique for each dataset config eg. bioasq10b_(source|bigbio)_[bigbio_schema_name]
+    #   - version: option = (SOURCE_VERSION |BIGBIO_VERSION)
```

Comment:
might want to add a space after the pipe or remove the space before it, but its not a big deal either way

Reply:
done!